### ISSUE TITLE
What dictates the Ubuntu version selection?

### ISSUE THREAD

**Issue by lucasfolino** [raspberrypi/rpi-imager#223]
I am looking to install Ubuntu Desktop 64 Bit LTS (20.04) on my Pi 4.  This versions is not in the Ubuntu version selector.  Is there a way to change this?  Could this selection be changes to include any supported version of Ubuntu.  I like using the LTS even though it is available for the server option it is not for the Desktop.   I could use some other third party tool to write the disk, but was curious how the list of Distros and versions are curated.

**Comment by maxnet** [comment]
May be better to ask Ubuntu.
They maintain their own distribution list.

(We link to: https://changelogs.ubuntu.com/raspi/os_list_imagingutility_ubuntu.json )

**Comment by lucasfolino** [comment]
I installed the Raspberry Pi Imager on Windows 1.6.1.  Where do these lists originate?  The install was from raspberrypi.org.  Is each supply you with a list of versions or is this internally managed?

**Comment by lurch** [comment]
ping @waveform80 in case he knows anything about this.

**Comment by waveform80** [comment]
Hi @lucasfolino -- there's a simple explanation for this: there is no 20.04 desktop image. Groovy (20.10) was the first version where we released an official Ubuntu desktop image. It's possible to install the various *-desktop packages on a 20.04 server install, but that's not a supported option (i.e. we don't test that in the labs and there's no support cycle associated with such a configuration). The first Ubuntu LTS desktop for the pi should (hopefully!) be 22.04.

**Comment by lucasfolino** [comment]
Thank you for the explanation.  Does that mean moving forward that the list will be supported versions?

**Comment by waveform80** [comment]
TL;DR: All currently supported versions, no, but all versions listed will be currently supported.

Longer explanation:

* The current release will be listed; at the moment that's hirsute (21.04) server (armhf & arm64), and desktop (arm64 only) images
* The latest LTS release will be listed; currently focal (20.04) server (armhf & arm64)
* The latest Core release will be listed; currently core 18 (armhf & arm64)

However, this is not the complete set of supported releases. For instance, groovy (20.10) is still supported until July 2021 and the prior LTS, bionic (18.04), is [supported until April 2023](https://wiki.ubuntu.com/Releases), but we don't list either in the RPi Imager JSON.

Obviously it's preferable from a support perspective to guide people to the latest releases, and we generally only back-port support for new hardware to the latest LTS (the Pi 400, for example, is supported on focal but not bionic). Hence, we want to list those images with the widest audience, and not clutter the interface with options that (although [still available](http://cdimage.ubuntu.com/releases/bionic/release/) and supported), have less relevance to most users.

**Comment by lucasfolino** [comment]
Ok so the Latest LTS will always be in the list.  And moving forward once there is a desktop supported version of an LTS for the Pi that will be in the list.  Thank you for the details.
